actionview/lib/action_view/renderer/abstract_renderer.rb

```diff
@@ -134,7 +134,7 @@ def initialize(format)
           @format = format
         end
 
-        def body; nil; end
+        def body; " "; end
```

Comment:
I think it's strange that rendering an empty collection should return `" "`, but that's the existing behavior, so I'll save that change for a separate PR. 😅